schema/partners.js

```diff
@@ -36,8 +37,7 @@ const Partners = {
       `,
     },
     type: {
-      type: new GraphQLList(GraphQLString),
-      description: 'Only return partners of the specified _type(s).',
+      type: new GraphQLList(PartnerClassification.type),
```

Comment:
can we just use classification to avoid the confusion? it has type as one of it's fields, so i don't believe it would be typetype if you look at the pattern. it isn't a graphqlobjectype or whatever.